File: VTI_Galaxy_BE/src/main/java/demo/services/interfaceClass/SeatRoomService.java
package demo.services.interfaceClass;

import demo.modal.dto.SeatRoomDto;
import java.util.List;

public interface SeatRoomService {
    SeatRoomDto getSeatRoomById(int id);

    List<SeatRoomDto> getSeatRoomsByRoomId(int roomId);
}

File: VTI_Glaxy_FE/src/services/seatRoomService.js
import axiosClient from "./axiosClient";

const seatRoomService = {
  fetchSeatRoomById(seatRoomId) {
    return axiosClient.get(`/getSeatRoomById?seatRoomId=${seatRoomId}`);
  },
  fetchSeatRoomByRoomId(roomId) {
    return axiosClient.get(`/getSeatRoomsByRoomId?roomId=${roomId}`);
  },
};

export default seatRoomService;
